### PR TITLE
Add a dark theme "v_material"

### PR DESCRIPTION
**Pull request by NoCLin**

### CODE REVIEW

File: src/resources/themes/v_material/v_material.palette [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+listview_item_selected_inactive_bg=@inactive_bg
+
+; Splitter.
+splitter_handle_bg=@border_bg
+
+; StatusBar.
+statusbar_fg=@main_fg
+statusbar_bg=@main_bg
+
+
```
Comment (tamlok) [inline src/resources/themes/v_material/v_material.palette]: I will remove this extra empty line.

File: src/resources/themes/v_material/v_material.palette [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+master_fg=#eeffff
+
+master_bg=#80cbc4
+
+base_fg=#80cbc4
+
+base_bg=#263238
+
+
```
Comment (tamlok) [inline src/resources/themes/v_material/v_material.palette]: Extra empty line.

File: src/resources/themes/v_material/v_material.mdhl [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+# [VNote] style for selected word highlight
+selected-word-background: dfdf00
+# [VNote] style for searched word highlight
+searched-word-background: 4db6ac
+# [VNote] style for searched word under cursor highlight
+searched-word-cursor-background: 66bb6a
+# [VNote] style for incremental searched word highlight
+incremental-searched-word-background: ce93d8
+
+
```
Comment (tamlok) [inline src/resources/themes/v_material/v_material.mdhl]: Extra empty line.

File: src/resources/themes/v_material/v_material.mdhl [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+REFERENCE
+foreground: b58900
+
+CODE
+foreground: C792EA
+font-family: Consolas, Monaco, Andale Mono, Monospace, Courier New
+
+EMPH
+font-style: italic
+
```
Comment (tamlok) [inline src/resources/themes/v_material/v_material.mdhl]: Extra empty line.

File: src/resources/themes/v_material/v_material.palette [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+[metadata]
+qss_file=v_material.qss
+mdhl_file=v_material.mdhl
+css_file=v_material.css
+codeblock_css_file=v_material_codeblock.css
+
+[phony]
+; Abstract color attributes.
+
+
```
Comment (tamlok) [inline src/resources/themes/v_material/v_material.palette]: extra empty line.

**Comment by tamlok** [comment]
Add drak style from @NoCLin.

**Comment by NoCLin** [comment]
Thanks, I'll pay more attention to it next time.

**Pull request merged**